### ISSUE TITLE
Deprecated time_series_19-covid-Confirmed.csv but no US State data in the Global confirmed file

### ISSUE THREAD

**Issue by allenbroadman** [CSSEGISandData/COVID-19#1534]
Hi. Thank you for all the efforts at providing this data.

time_series_19-covid-Confirmed.csv has been deprecated and we are instructed in the Readme file to use the Global confirmed file instead.

I have been relying on time_series_19-covid-Confirmed.csv for United States individual state data which is not contained in the Global confirmed file.

Where are you putting the USA state data?

**Comment by davidlyon3** [comment]
These new files is nothing short of a DISASTER

**Comment by smillerd** [comment]
This is a big bummer. I was making decent progress on building my monitoring suite on this data for my US state. I'm looking for fresh sources, but they all seem to trace back to this repo! What a mess.

**Comment by cjparisi** [comment]
refer to issue: #1250

**Comment by soaring52** [comment]
Please add US states (and all other states/provinces that might be missing). Many of us depend on the full set of data. 
Also, why the "recovered" cases are not reported? It would be good to bring them back. I am aware of issue: #1250!!!
The people age for all 3 cases (confirmed, deaths, and recovered) would significantly add the value to the "cleaned" data sets.
Thanks for all the effort and time putting this together
_i

**Comment by soaring52** [comment]
**### Canada  States**: ['Alberta' 'British Columbia' 'Grand Princess' 'Manitoba' 'New Brunswick'
 'Newfoundland and Labrador' 'Nova Scotia' 'Ontario'
 'Prince Edward Island' 'Quebec' 'Saskatchewan' **'Diamond Princess**'
 '**Recovered**']
The highlighted looks like a bug

**Comment by josephsdavid** [comment]
Agreed with this entirely, the sudden data change is outrageous

**Comment by Tolga28A** [comment]
Dear JH researchers,

I can't tell how much I appreciate this amazing work you've been putting in these dark times. I see that I am not the only one who has started building models based on your US-territory data until two days ago, and our deliverables might impact the progression of this disease in global scale (i.e. I am personally a data scientist in one of the largest global pharma companies). 

I personally relied on the fact that you said you were going to release a separate dataset for US; however, we can see neither an explanation nor any data. And as you'd imagine, time is super critical especially predictions like disease spread forecast and we can see the US data updated in the dashboard. So please could someone at least give an update about where you are at publishing US data again? If this is not going to happen for some reason, then we can go and try to find other data sources instead of losing time here...

**Comment by AndBurns** [comment]
Concur with above

**Comment by mrwallison** [comment]
This seems to have been a step backward and a loss of granularity, especially for the united states.

**Comment by cognospaul** [comment]
Until the granular data files come back here, I've been sourcing my data from usafacts.org:
https://usafacts.org/visualizations/coronavirus-covid-19-spread-map/

I've seen a few issues with the data, such as deaths in Michigan disappearing. Overall I think it's a decent source.

**Comment by Tolga28A** [comment]
> Until the granular data files come back here, I've been sourcing my data from usafacts.org:
> https://usafacts.org/visualizations/coronavirus-covid-19-spread-map/
> 
> I've seen a few issues with the data, such as deaths in Michigan disappearing. Overall I think it's a decent source.

This is great, thank you so much!

**Comment by mskymoore** [comment]
Agreed.  Would really appreciate the granularity for US states especially.

**Comment by cognospaul** [comment]
also, another tidbit. If you want to include county population, you can pull that from:

https://www.census.gov/data/datasets/time-series/demo/popest/2010s-counties-total.html

The join would be on usafacts.statesFIPS to census.STATE and usafacts.countyFIPS to (if(census.COUNTY=0) then (0) else (census.COUNTY + (census.STATE*1000)))

**Comment by rks125** [comment]
I recreated the Time Series CSV files for Confirmed, Recovered, and Deaths with US State data:

https://www.soothsawyer.com/john-hopkins-time-series-data-confirmed-case-csv-after-march-22-2020/

I also included the PowerQuery tool I created to make those CSV files.  This should hold you over.  I currently plan on refreshing the CSV files by 5:15pm PT daily, until no longer necessary.

**Comment by allenbroadman** [comment]
Thanks very much rks125. This collectively saves a lot of duplicated effort for all of us basing our models on the deprecated file format, until Johns Hopkins gets this working.

**Comment by soaring52** [comment]
Please be aware that in the **"deaths"** data set the country **"The West Bank and Gaza"** is **missing**. If you want to correlate number of deaths vs. global reported cases you need to remove the "The West Bank and Gaza" from the "global" dataset, first.
It is great that others provide alternatives for the "US states" data, but it would be still the best if we can have access to the (CDC approved) official single and validated dataset for all of us to use.
Thanks all.

**Comment by pybokeh** [comment]
FYI, time series data for US has been released: https://github.com/CSSEGISandData/COVID-19/issues/1250

**Comment by josephsdavid** [comment]
Thank you!!

________________________________
